release-notes/swan-lake-beta1-release-note.md

```diff
@@ -61,13 +61,56 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Beta1](https://github
 
 ### Standard Library Updates
 
-#### Improvements
+#### New features
 
-##### `graphql` Package
+##### GraphQL Package
+- Add declarative auth configurations
+- Support optional types as inputs
+- Support returning distinct service object unions
+- Support inline fragments
+- Support enums as input values
 
- - Allow enum as an input parameter.
+#### Improvements
 
-#### Bug Fixes
+##### GraphQL Package
+- Improved introspection validation and execution
+- Added missing fields in GraphQL types
+- Compiler plugin improvements to validate inputs and return types
+- Use included record parameters instead of record type in listener initialization
+
+##### HTTP Package
+- Improve `http:Client` remote methods to support contextually expected type inference.
+- Change configuration parameters of listeners and clients to included record parameters.
+
+##### WebSubHub Package
+- Include Auth Configuration to WebSubHub PublisherClient configuration
+
+#### Bug fixes
+
+##### GraphQL Package
+- Allow enums as input parameters in resource functions
+- Fix service hanging when service type array is returned
+- Fix stack overflow when types have recursive references
+- Fix incorrect values returning from `enumValues` field
+- Fix validating non-existing fields in service types
+- Fix invalidation of queries when a defaultable parameter is not provided
+- Fix `LIST` of `NON_NULL` types not showing correctly in introspection
+- Fix incorrect location value in fragments when returning an error
+- Fix query depth validation ignoring the fragment depth
+- Fix service crashing when unnamed union type is returned
+- Fix hanging of the service when an enum array is returned
+
+##### HTTP Package
+- Prepend resource path argument of client method with a forward slash if not added.
+
+##### WebSub Package
+- Fix the listener initialization with inline configs compiler plugin error
+
+##### WebSubHub Package
+- Fix the listener initialization with inline configs compiler plugin error
+
+##### TCP Package
+- Fix the secure listener initialization failure when ciphers are not configured.
 
 To view bug fixes, see the [GitHub milestone for Swan Lake Beta1](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%22Swan+Lake+Beta1%22+label%3AType%2FBug).
```

Comment:
```suggestion
To view bug fixes, see the [GitHub milestone for Swan Lake Beta1](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%22Swan+Lake+Beta1%22+label%3AType%2FBug).
```
Just keeping this paragraph should be enough for bug fixes.

Reply:
So shall I remove the `Bug Fixes` section?